### PR TITLE
Create webpage for step3 of meeting creation.

### PR DESCRIPTION
**Pull request by theresenieva**
Create webpage for step 3 (Choose meeting locations) of the meeting creation process.

### CODE REVIEW

**Review by vivianho18 — COMMENTED** [review]
looks good :)

**Review by anna-giang — APPROVED** [review]
LGTM 👍

File: GoMeet/src/main/webapp/create-meeting-step3.html [hunk 1]
```diff
@@ -4,7 +4,20 @@
   <head>
     <meta charset="UTF-8" />
     <title>Create Meeting</title>
+    <script src="js/load-map.js"></script>
+    <script src="js/meeting-location-dao.js"></script>
+    <script src="js/interactive-map.js"></script>
+    <script src="js/error-messages.js"></script>
```
Comment (BipBopBoop) [inline GoMeet/src/main/webapp/create-meeting-step3.html]: nit: generally good to order things alphabetically when declaring/importing them.

**Review by BipBopBoop — APPROVED** [review]
lgtm % 1 nit.

**Pull request merged**